homeassistant/components/zeversolar/diagnostics.py

```diff
@@ -0,0 +1,61 @@
+"""Provides diagnostics for Zeversolar."""
+
+from typing import Any
+
+from zeversolar import ZeverSolarData
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntry
+
+from .const import DOMAIN
+from .coordinator import ZeversolarCoordinator
+
+TO_REDACT = {"serial_or_registry_id", "registry_key", "serial_number"}
```

Comment:
if you redact all of those, how would the user (with multiple  devices) know which device it is?

Reply:
Yes, you are right. I got confused by the naming.
Looking onto my own Inverter, I checked that these keys are not the keys used for registration on the ZeverCloud.
So it is no security issue to show them, I have removed the complete redaction.

Just for clarification:
"SerialNumber" means the serial number of the inverter.
"DeviceId" points to a "device", that groups up to 3 inverters. The "device" is registered into HA by its IP address.